Burkina_Faso/BFA_transform_load_dlt.py

```diff
@@ -235,9 +131,7 @@ def boost_gold():
                        'admin1',
                        'admin2',
                        'geo1',
-                       'func',
-                       'econ_sub',
-                       'econ'
```

Comment:
@nihaas21 This doesn't look right because we definitely want to keep econ_sub and econ processing code. I'm closing this PR. Please pull the main branch and create a new branch from the latest code, apply your changes and create a new PR.